src/mcpbr/infrastructure/k8s.py

```
@@ -0,0 +1,1190 @@
+            input_data: Optional string piped to stdin.
+            timeout: Command timeout in seconds.
+            check: If True, raise on non-zero exit code.
+
+        Returns:
+            CompletedProcess result.
+
+        Raises:
+            RuntimeError: When check=True and the command fails.
+        """
+        full_cmd = self._kubectl_base() + args
+        result = subprocess.run(
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)